scripts/zkai_attention_kv_stwo_native_d8_bounded_weighted_proof_input.py

```
@@ -0,0 +1,518 @@
+    "final_kv_cache_commitment",
+    "outputs_commitment",
+    "statement_commitment",
+    "non_claims",
+)
+
+
+class AttentionKvBoundedWeightedInputError(ValueError):
+    pass
+
+
+def _load_source_module() -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_load_source_module`

(ANN401)